### PR TITLE
Enhancements to Native Full Withdrawal Flow, Minor Delegation wording update

### PR DESCRIPTION
**Pull request by wesfloyd**

### CODE REVIEW

**Comment by wesfloyd** [comment]
Hi @Chris-Moller per our [discussion here](https://github.com/Layr-Labs/ui-monorepo-v2/pull/848#issuecomment-2201455144) and to avoid recent proof gen errors with full withdrawals, I've front loaded asking the user to fully exit BEFORE they queue withdrawal. I've integrated the new flow and made significant changes to the flow. Can I get your help to review and confirm accuracy?

Also, in particular:

> **Step 8:** Observe the Unstaked Beacon Chain balance has increased after the escrow period. The Execution Chain amount should have increased by the amount of those exited validators.
Is this line correct? I actually don't know how the UI is intended to behave at this point in the process.

Thank you

File: docs/eigenlayer/restaking-guides/0-restaking-user-guide/native-restaking/withdraw-from-eigenlayer/full-withdrawal.md [hunk 1]
```diff
@@ -4,41 +4,34 @@ sidebar_position: 2
...
 
-:::tip
-Exits from the beacon chain can be completed in independently of (in parallel, at the same time as) queuing a Full Withdrawal from EigenLayer.  This optimization may reduce your overall (end to end) time required to withdraw Native restaked funds from EigenLayer.
-:::
+**Step 1:** Ensure you have [repointed your validator's withdrawal credentials](../create-eigenpod-and-set-withdrawal-credentials/repointing-a-validators-withdrawal-credentials.md) to your EigenPod.
 
-:::warning
-Ensure you have [repointed your withdrawal credentials](../create-eigenpod-and-set-withdrawal-credentials/repointing-a-validators-withdrawal-credentials.md) to your EigenPod before proceeding.
-:::
+**Step 2:** Fully exit your validator from the beacon chain. You may monitor its activity via beaconcha.in/validator/[yourvalidatorid] .
```
Comment (Chris-Moller) [inline docs/eigenlayer/restaking-guides/0-restaking-user-guide/native-restaking/withdraw-from-eigenlayer/full-withdrawal.md:12]: nit: [yourvalidatorindex] or [yourvalidatorpublickey]

File: docs/eigenlayer/restaking-guides/0-restaking-user-guide/native-restaking/withdraw-from-eigenlayer/full-withdrawal.md [hunk 2]
```diff
@@ -4,41 +4,34 @@ sidebar_position: 2
...
-At this point you must exit your beacon chain validators prior to continuing. The validators must have completely exited from the validator queue before continuing. You may use [beaconcha.in](https://beaconcha.in) as a reference during this process.
-:::
-
-After your validators have exited, the Execution Chain amount should increase by the amount of those exited validators.
+**Step 8:** Observe the Unstaked Beacon Chain balance has increased after the escrow period. The Execution Chain amount should have increased by the amount of those exited validators.
 
 The Redeposit button is available at this point to allow the user to Restake back into EigenLayer in case the withdrawal was queued by mistake.
 
-**Step 5:** Once your validator has been exited click Withdraw to complete your deposit. **Sign** the transaction using your Web3 wallet.
+**Step 9:** Once your validator has been exited click Withdraw to complete your deposit. **Sign** the transaction using your Web3 wallet.
```
Comment (Chris-Moller) [inline docs/eigenlayer/restaking-guides/0-restaking-user-guide/native-restaking/withdraw-from-eigenlayer/full-withdrawal.md:34]: Maybe change the wording, we already told them the validator should be exited before queueing so this is confusing. Maybe "Once your funds clear the escrow". 

Also it should he complete your withdrawal, not deposit right?

File: docs/eigenlayer/restaking-guides/0-restaking-user-guide/restaker-delegation/redelegation-process.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ Please follow the recommended steps below carefully to avoid a "partially delega
 ![undelegate button](/img/restake-guides/delegate-3.png)
 
 
-**Step 2:** **Confirm** the two transactions in your Web3 wallet.
+**Step 2:** **Confirm** the Undelegate transaction in your Web3 wallet.
```
Comment (Chris-Moller) [inline docs/eigenlayer/restaking-guides/0-restaking-user-guide/restaker-delegation/redelegation-process.md:24]: 👍

File: docs/eigenlayer/restaking-guides/0-restaking-user-guide/native-restaking/withdraw-from-eigenlayer/full-withdrawal.md (hunk 1 above)
Comment (wesfloyd) [reply docs/eigenlayer/restaking-guides/0-restaking-user-guide/native-restaking/withdraw-from-eigenlayer/full-withdrawal.md:12]: Thanks, that's a good nit

File: docs/eigenlayer/restaking-guides/0-restaking-user-guide/native-restaking/withdraw-from-eigenlayer/full-withdrawal.md (hunk 2 above)
Comment (wesfloyd) [reply docs/eigenlayer/restaking-guides/0-restaking-user-guide/native-restaking/withdraw-from-eigenlayer/full-withdrawal.md:34]: Good catch! That was a holdover from the old ordering I forgot to fix. Updated now.

**Review by dabit3 — APPROVED** [review]

**Pull request merged**